expose/patch-ingress.yml

```diff
@@ -11,3 +11,17 @@ metadata:
 spec:
   expose:
     type: ingress  # Spinnaker is exposed using ingress. Available options: service, ingress
+  config:
+    security:
+      apiSecurity:
+        overrideBaseUrl: https://spinnaker.mycompany.com/api  # Public API (Gate) url. Using the same hostname for deck and gate only works when exposing spinnaker with ingress.
```

Comment:
If you specify `type: ingress` then you shouldn't need to specify all this. That's pretty much the only thing `type: ingress` does. The readiness probe is transformed from an `exec` to an `httpGet`. Although it currently doesn't change the `healthEndpoint`.